app/src/main/res/values/strings_settings.xml

```diff
@@ -35,84 +35,84 @@
 
     <!-- Search -->
     <string name="search_icon_title">Search</string>
-    <string name="search_hint">Search applications</string>
+    <string name="search_hint">Search for apps</string>
 
     <!-- Other settings -->
     <string name="set_transparent_status_bar">Transparent status bar</string>
     <string name="set_transparent_status_bar_hint">When favorites are closed</string>
-    <string name="set_hide_menu_button">Hide the menu button</string>
-    <string name="set_hide_menu_button_help">It stays accessible using the launcher icon</string>
+    <string name="set_hide_menu_button">No 🔧 menu button</string>
+    <string name="set_hide_menu_button_help">Always accessible from the launcher app icon</string>
     <string name="set_icon_pack">Use an installed icon pack</string>
-    <string name="set_icon_pack_none">None (use the real icons)</string>
-    <string name="set_hide_app_names">Hide applications names</string>
-    <string name="set_hide_app_names_help">Folder names will still be displayed</string>
-    <string name="set_remove_padding">Remove padding when names are hidden</string>
-    <string name="set_remove_padding_help">May produce unexpected results with long folders names</string>
-    <string name="set_force_portrait">Force portrait mode</string>
-    <string name="set_force_portrait_help">The launcher will not rotate even if automatic rotation is enabled</string>
+    <string name="set_icon_pack_none">None (use default icons)</string>
+    <string name="set_hide_app_names">No app names</string>
+    <string name="set_hide_app_names_help">Folder names are still shown</string>
+    <string name="set_remove_padding">No padding when app names are hidden</string>
+    <string name="set_remove_padding_help">Can cause problems with long folders names</string>
+    <string name="set_force_portrait">Forced portrait-mode</string>
+    <string name="set_force_portrait_help">Prevents the launcher rotating even if auto-rotation is on</string>
     <string name="set_always_show_favorites">Always show favorites</string>
-    <string name="set_immersive_mode">Use immersive mode</string>
-    <string name="set_immersive_mode_help">Hide system bars (long press on home screen to force if needed)</string>
-    <string name="set_reverse_interface">Reverse interface</string>
-    <string name="set_reverse_interface_help">Display the favorites panel on bottom</string>
-    <string name="set_disable_app_drawer">Disable app drawer</string>
-    <string name="error_disable_app_drawer_not_safe">App drawer enabled to prevent you from losing menu access</string>
-    <string name="error_appfilter_not_found">The appfilter.xml file was not found in the selected icon pack! (%1$s)</string>
+    <string name="set_immersive_mode">Immersive mode</string>
+    <string name="set_immersive_mode_help">No system bars. (Can be changed by long-pressing the home screen.)</string>

```

Comment:
```suggestion
    <string name="set_immersive_mode_help">No system bars. (Can be forced by long-pressing the home screen.)</string>
```
Maybe the remark in parenthesis is a bit less clear now... (as it is not a toggle, it only hides system bars)

Here is the idea: after the user has triggered the display of the system bars (ex: by opening and closing the notification drawer), the launcher doesn't not come back immediately in immersive mode. It will come back to it only after an app or something else (ex: lock screen) is displayed and when return to the home screen (this is to avoid the system bars playing hide-and-seek if the user needs them).
=> If the user wants to force the return in immersive mode without opening something else in between, (s)he can simply long press the home screen and it will hide again the system bars.